lib/yext/api/utils/configuration.rb

```diff
@@ -55,6 +55,23 @@ module Utils
       #
       # sandbox
       #   Boolean that indicates if the gem should use the production or sandbox URL.
+      #
+      # default_callback_processor
+      #   The default class to be used to process webhook messages that are called.
+      #   The code will warn about any missing webhook messages in a default processor.
+      # set_callback_processor(callback_function_name, processor)
```

Comment:
I expected something a little less opinionated. Similar (simpler?) to the stripe call back setup which uses ActiveSupport::Notifications

Reply:
I did something and I won't say I was thrilled with it.

This probably needs a major whack upside the head - but I didn't know about `ActiveSupport::Notifications`, so that is probably what I should use.

Again, changing this may be a future PR though.